src/lib/hooks/__tests__/useLazyQuery.test.tsx

```diff
@@ -97,4 +97,73 @@ describe("useLazyQueryWithPagination", () => {
       expect(result.current[1].data).toBe(mockFormattedDataString);
     });
   });
+
+  describe("api call cancellation", () => {
```

Comment:
should we include a test where pass our abort controller?

Reply:
these test cases are for hooks, where we don't accept abortController